.github/workflows/conformance.yaml

```diff
@@ -549,6 +549,26 @@ jobs:
           tests-path: validating-admission-policy-reports
           upgrade-scenario: ${{ matrix.upgrade }}
 
+  image-verification-policies:
```

Comment:
should we already align the naming to `image-validation-policies`?

Reply:
Good point, renamed!